lib/game.rb

```diff
@@ -0,0 +1,67 @@
+require_relative './score'
+
+class Game
+  attr_reader :throws, :score, :frame_count, :last_frame
+
+  MAX_FRAME = 10
+
+  def initialize(score = Score.new)
+    @throws = []
+    @frame_complete = true
+    @score = score
+    @frame_count = 1
+    @last_frame = []
+  end
```

Comment:
I'd be interested to play this in irb as I can't quite follow how score.rb and game.rb link up